### DISCUSSION TITLE
Highlight inline code

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#16118] [Ideas] [answered]
Small idea, but it would be nice if inline code was highlighted with a background to make it standout more.

Before:

![CleanShot 2024-11-14 at 13 36 33@2x](https://github.com/user-attachments/assets/6e231193-6605-47c7-95eb-75fd3576b5a0)

After:

![CleanShot 2024-11-14 at 13 36 06@2x](https://github.com/user-attachments/assets/9502be46-e2b5-4441-bd50-0e776b27d752)

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion! We made that change for Craft 5.6 (#16141).

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out with that change 🎉
